### PR TITLE
commercial-consent-global-banner A/B test

### PR DESCRIPTION
**Pull request by GHaberis**
## What does this change?

This PR replaces the A/B test `ab-commercial-consent-global-no-scroll` with the multi-variant A/B test `ab-commercial-consent-global-banner`.

The new `ab-commercial-consent-global-banner` test (ID: `CommercialConsentGlobalBanner`) includes the following variants:

`control`: no banner
`regularVariant`: regular banner
`noScrollVariant`: regular banner + inability to scroll the page through the banner
`tallVariant`: regular banner + increase in banner height
`animationVariant`: regular banner + an animation that shows the banner siding up from the bottom of the screen after a brief delay
`floatingVariant`: floating box banner

## Screenshots

**regularVariant - mobile**
regularVariant-mobile

**regularVariant - desktop**
regularVariant-desktop

**tallVariant - mobile**
tallVariant-mobile

**tallVariant - desktop**
tallVariant-desktop

**animationVariant - mobile** (wait for it, 🥁 3, 2, 1...)
animatedVariant-mobile

**floatingVariant - mobile**
floatingVariant-mobile

**floatingVariant - desktop**
floatingVariant-desktop

## What is the value of this and can you measure success?

Test whether changes to Consent Banner increases proportion of non-EU users who interact with it

### Tested

- [x] Locally
- [ ] On CODE (optional)

### CODE REVIEW

File: static/src/javascripts/projects/common/modules/experiments/tests/commercial-consent-global-banner.js [hunk 1]
```diff
@@ -1,20 +1,20 @@
...
-    audience: 0.02,
+        'Test whether changes to Consent Banner increases proportion of non-EU users who interact with it',
+    audience: 0.03,
     audienceOffset: 0.1,
     successMeasure: 'Users outside of the EU interact with the consent banner',
     audienceCriteria: 'all users',
     dataLinkNames: '',
     idealOutcome:
-        'More non-EEA users interact with the consent banner when we prevent the ability to scroll underlying content through the banner on mobile',
+        'More non-EEA users interact with the consent banner when we alter ir',
```
Comment (adamnfish) [inline static/src/javascripts/projects/common/modules/experiments/tests/commercial-consent-global-banner.js]: Typo here? `ir` -> `it`. Might be worth wording this as something like `fewer non-EEA users ignore the altered banner, which reduces the number of pageviews with unset conset`

File: static/src/stylesheets/module/site-messages/_first-pv-consent.scss [hunk 2]
```diff
@@ -194,3 +194,65 @@ $btn-height: ($gs-baseline*3.5);
...
+    .site-message--first-pv-consent__link {
+        margin-top: $gs-baseline * 1.5;
+        margin-left: 0;
+    }
+    .site-message--first-pv-consent__button {
+        margin-right: $gs-gutter * .5;
+    }
+
+.site-message--first-pv-consent--animationVariant {
+    transform: translateY(100%) translateZ(0); // translateZ to force GPU
```
Comment (adamnfish) [inline static/src/stylesheets/module/site-messages/_first-pv-consent.scss]: Might be worth checking if we see a change in the crash rate , especially on fronts. Just mentioning this to dotcom is probably enough?

**Review by adamnfish — COMMENTED** [review]
Looks ok to me, although I confess I am no expert!

File: static/src/stylesheets/module/site-messages/_first-pv-consent.scss (hunk 2 above)
Comment (GHaberis) [reply static/src/stylesheets/module/site-messages/_first-pv-consent.scss]: I'll ask, I'm aware of `translateZ(0)` being a problem when applying the rule to huge elements (thousands of px in size) or elements positioned thousands of pixels off screen, but this is a relatively small element, and I checked and found this technique applied in numerous places in `frontend`.

**Review by adamnfish — APPROVED** [review]
LGTM

**Pull request merged**